### ISSUE TITLE
Grid Accessibility: Custom Cell elements do not receive focus and cannot be triggered with keyboard

### ISSUE THREAD

**Issue by jsmapr1** [ag-grid/ag-grid#3359]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```

Reproducible example:

See custom button cell in documentation:
https://www.ag-grid.com/javascript-grid-cell-rendering-components/#example-rendering-using-react-components

Tabbing to the cell doesn't allow the button to take focus and does not respond to key down events (see gif)

![grid accessibility](https://user-images.githubusercontent.com/2388943/64437465-5289a380-d094-11e9-9214-a963e1edad85.gif)

**Current behavior**
Custom cell elements do not receive focus. As a result, buttons can not be triggered from the keyboard.

**Expected behavior**
When I tab to a custom cell a button in a custom cell will be able to receive focus.

**Please tell us about your environment:**
Occurs on the documentation site. Accessed with mac osx and chrome (see agent below)

* **ag-Grid version:** 21.1.1

* **Browser:**
"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.100 Safari/537.36"

"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.1.2 Safari/605.1.15"

* **Language:** ES2019

**Comment by davis4acca** [comment]
Hi,

Your ticket has been flagged as in_zendesk, it means that we recognize this as an issue worth investigating so we have moved it to our official support channel for customers (zendesk)

We will carry on there with the investigation and we will update you as soon as possible.

Thanks

**Comment by AhmedAGadir** [comment]
Hi,

In our knowledge base, we have some examples that I think are relevant. Can you have a look at this section

https://next.plnkr.co/edit/AixUUlhNeFSWYRFw?preview

We believe this should get you kickstarted, but if you don't think this covers your use case, please feel free to follow up on this thread.

**Comment by dwilt** [comment]
@jsmapr1 were you ever able to solve this? we're using React and running into this as well. We can't seem to get `getCellRendererInstances` to return any instances back..
